index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
+        <li><input type="text" placeholder="Full Name" maxlength="30" required></li>
+        <li><input name="Email" type="email" id="email" placeholder="Email address" required></li>
+        <li><textarea name="Comment" placeholder="Enter text here" rows="5" maxlength="500" required></textarea></li>
+      </ul>
+      <button class="submit">Submit</button>
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure the `button` has the `type` attribute and must be set to `type="submit"`.